code/modules/organs/external/_external.dm

```diff
@@ -272,42 +265,74 @@
 			E.update_icon()
 			return
 
+	//Remove sub-limbs
+	if(istype(W,/obj/item/circular_saw) && LAZYLEN(children) && try_saw_off_child(W, user))
```

Comment:
Please check for `TOOL_SAW` (ideally use the tool procs for the interaction).

Reply:
Done